src/apps/base-images/gdal/Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG TAG=latest
+#FROM geoservercloud/gs-cloud-base-jre:$TAG as builder
+FROM eclipse-temurin:11-jdk as builder
+
+LABEL maintainer="GeoServer PSC <geoserver-users@lists.sourceforge.net>"
+
+ENV GDAL_VERSION 3.8.4
+ENV GDAL_PATH /usr/share/gdal
+ENV GDAL_DATA $GDAL_PATH/3.8.4
+ENV PATH $GDAL_PATH:$PATH
+ENV LD_LIBRARY_PATH $LD_LIBRARY_PATH:/usr/lib/jni:/usr/share/java:/usr/local/lib/:/usr/lib/
+ENV ANT_VERSION 1.9.16
```

Comment:
Is this environment variable actually needed? If so, it might get out of sync with the required `ant` version since it might change in the package repository.

Reply:
yeap, ANT_VERSION is not required (removing it)